src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -67,35 +67,25 @@ const defaultProps = {
     },
     priorityMode: CONST.PRIORITY_MODE.DEFAULT,
 };
-
-class SidebarLinks extends React.PureComponent {
-    constructor(props) {
-        super(props);
-
-        this.showSearchPage = this.showSearchPage.bind(this);
-        this.showSettingsPage = this.showSettingsPage.bind(this);
-        this.showReportPage = this.showReportPage.bind(this);
-
-        if (this.props.isSmallScreenWidth) {
-            App.confirmReadyToOpenApp();
-        }
+function SidebarLinks(props) {
```

Comment:
The style guideline has changed and we now destructure props.
We also apply all default values in the destructuring.

Read more here: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring